test/ansible/build/Dockerfile

```diff
@@ -1,5 +1,8 @@
-FROM quay.io/operator-framework/ansible-operator:dev
+FROM quay.io/operator-framework/ansible-operator
```

Comment:
Should not here it still being dev tag? 
To use the image  built in the CI?

Reply:
ah whoops, good catch that was a change for local testing